apps/fluent-tester/src/TestComponents/RadioGroup/RadioGroupTest.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { View } from 'react-native';
-import { RadioButton, RadioGroup, Separator } from '@fluentui/react-native';
+import { Separator } from '@fluentui/react-native';
```

Comment:
I believe we should make a new Experimental RadioGroup test page (I think that's been the pattern as we develop updated Experimental controls).

The reason being it's going to take some time to get:

1. The new control fully developed
2. Partners consuming the new control

Also, we want our E2E testing to continue to test our current RadioGroup control. Partners are still using it and we need to validate we don't regress it.

Reply:
Makes sense, creating a new RadioGroupExperimental test page.